forum/qa-theme/SnowFlat/js/selectLanguage.js

```diff
@@ -0,0 +1,111 @@
+                        break;
+                        case 'PHP, Symfony, Zend':
+                            CKEDITOR.config.syntaxhighlight_lang = 'php';
+                        break;
+                        case 'SQL, bazy danych':
+                            CKEDITOR.config.syntaxhighlight_lang = 'sql';
+                        break;
+                        case 'C# i .NET':
+                            CKEDITOR.config.syntaxhighlight_lang = 'csharp';
+                        break;
+                        case 'Java':
+                            CKEDITOR.config.syntaxhighlight_lang = 'java';
+                        break;
+                        case 'Python, Django':
+                            CKEDITOR.config.syntaxhighlight_lang = 'python';
+                        break;
+                        default:
+                            CKEDITOR.config.syntaxhighlight_lang = 'plain';
+                        break;
+                    }    			
+                }    
+            });
+        } else if ( location.href.includes( "edit" ) ) {
+                codeBlock.addEventListener( "click", function() {
+                const questionEditCategory_1 = $('#q_category_1 option:selected');
+                if(questionEditCategory_1.text() === 'Programowanie'){
+                    const questionEditCategory_2 = $('#q_category_2 option:selected');
+                    switch(questionEditCategory_2.text()){
+                        case 'HTML i CSS':
+                            CKEDITOR.config.syntaxhighlight_lang = 'xml';
+                        break;
+                        case 'C i C++':
+                            CKEDITOR.config.syntaxhighlight_lang = 'cpp';
+                        break;
+                        case 'JavaScript, jQuery, AJAX':
+                            CKEDITOR.config.syntaxhighlight_lang = 'jscript';
+                        break;
+                        case 'PHP, Symfony, Zend':
+                            CKEDITOR.config.syntaxhighlight_lang = 'php';
+                        break;
+                        case 'SQL, bazy danych':
+                            CKEDITOR.config.syntaxhighlight_lang = 'sql';
+                        break;
+                        case 'C# i .NET':
+                            CKEDITOR.config.syntaxhighlight_lang = 'csharp';
+                        break;
+                        case 'Java':
+                            CKEDITOR.config.syntaxhighlight_lang = 'java';
+                        break;
+                        case 'Python, Django':
+                            CKEDITOR.config.syntaxhighlight_lang = 'python';
+                        break;
+                        default:
+                            CKEDITOR.config.syntaxhighlight_lang = 'plain';
+                        break;
+                    }   		
+                }
+            });
+	    } else {
+            codeBlock.addEventListener( 'click', function() {

```

Comment:
Jeśli piszesz w ES6+ (a staramy się to przepisać z ES5 do góry) i nie jest Ci potrzebny zmienny kontekst **this**, to Arrow Function jak znalazł. :)